explorer/Explorer.tsx

```diff
@@ -336,10 +339,13 @@ export class Explorer
         this.updateGrapherFromExplorer()
 
         // preserve the previous tab if that's still available in the new view;
-        // and use the first tab otherwise
+        // and use the first tab otherwise, ignoring the table
+        const tabsWithoutTable = this.grapher.availableTabs.filter(
+            (tab) => tab !== GrapherTabOption.table
+        )
         newGrapherParams.tab = this.grapher.availableTabs.includes(previousTab)
             ? previousTab
-            : this.grapher.availableTabs[0]
+            : tabsWithoutTable[0] ?? GrapherTabOption.table
```

Comment:
Note for the reviewers: Explorers persist the current tab when switching charts. If the current tab is not available, it used to fallback to the first tab in Grapher's `availableTabs` property which used to be the chart or the map, but in the new designs is the table. That's not usually what we want, so here I explicitly ignore the table to make sure that we get either the map or the chart if the persisted tab is not available.